internal/apps/dop/endpoints/endpoints.go

```diff
@@ -462,7 +463,7 @@ func (e *Endpoints) Routes() []httpserver.Endpoint {
 
 		{Path: "/api/iterations", Method: http.MethodPost, Handler: e.CreateIteration},
 		{Path: "/api/iterations/{id}", Method: http.MethodPut, Handler: e.UpdateIteration},
-		{Path: "/api/iterations/{id}", Method: http.MethodDelete, Handler: e.DeleteIteration},
+		{Path: "/api/iterations/{id}/{onlyIteration}", Method: http.MethodDelete, Handler: e.DeleteIteration},
```

Comment:
this change will let original proxy invalid.

Reply:
ok，modified